index.html

```diff
@@ -0,0 +1,191 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
```

Comment:
- [ ]  I think it's better if you phrased the comments politely, as suggestions instead of orders. For now, Instead of `you should`, you can use ` I think you should add an HTML ...`.